numpy/lib/function_base.py

```diff
@@ -1726,7 +1726,9 @@ def place(arr, mask, vals):
 def disp(mesg, device=None, linefeed=True):
     """
     Display a message on a device.
-
+    
+    ..deprecated:: 1.17.0
+ 
```

Comment:
This shows up when someone does `help(disp)`, and should have text as well. See [this use](https://github.com/numpy/numpy/blob/v1.16.2/numpy/core/_add_newdocs.py#L1046) for instance.

`disp()` is deprecated, use `print()` instead